CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Improvements
 
 #### Dependency Upgrade
+* edit methods, now accept visitors as vararg or lambda.
```

Comment:
Umm, shouldn't this go in `Improvements` section?

Reply:
ohk, sorry. Just noticed we bumped Sundrio to `0.23.1`

Reply:
I think it'd be nice to add examples of each new modalities so that users can see the improvements directly.